changelog/changes/2024-07-26-landlock.md

```diff
@@ -0,0 +1 @@
+- Landlock is now enabled for programs to sandbox themselves.
```

Comment:
can you please add a PR ref here similar to https://github.com/flatcar/scripts/commit/7d4917d67cb76e50191eeb6395b083f224b02b78#diff-b1db1840bbe6435ffd261eeff937e5e0ade3e082a99ed9dff6affc044b92a22dR1 ? Also, I would suggest adding some context `- Linux kernel security module 'landlock' is now enabled for programs to be able to sandbox themselves  ([flatcar/scripts#2158](https://github.com/flatcar/scripts/pull/2158))`